x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/sequence/PITFailureTests.java

```diff
@@ -0,0 +1,231 @@
+        ) {
+            CircuitBreaker eqlCircuitBreaker = service.getBreaker(CIRCUIT_BREAKER_NAME);
+            EqlConfiguration eqlConfiguration = new EqlConfiguration(
+                new String[] { "test" },
+                org.elasticsearch.xpack.ql.util.DateUtils.UTC,
+                "nobody",
+                "cluster",
+                null,
+                emptyMap(),
+                null,
+                TimeValue.timeValueSeconds(30),
+                null,
+                123,
+                "",
+                new TaskId("test", 123),
+                new EqlSearchTask(
+                    randomLong(),
+                    "transport",
+                    EqlSearchAction.NAME,
+                    "",
+                    null,
+                    emptyMap(),
+                    emptyMap(),
+                    new AsyncExecutionId("", new TaskId(randomAlphaOfLength(10), 1)),
+                    TimeValue.timeValueDays(5)
+                ),
+                x -> emptySet()
+            );
+            IndexResolver indexResolver = new IndexResolver(esClient, "cluster", DefaultDataTypeRegistry.INSTANCE, () -> emptySet());
+            EqlSession eqlSession = new EqlSession(
+                esClient,
+                eqlConfiguration,
+                indexResolver,
+                new PreAnalyzer(),
+                new PostAnalyzer(),
+                new EqlFunctionRegistry(),
+                new Verifier(new Metrics()),
+                new Optimizer(),
+                new Planner(),
+                eqlCircuitBreaker
+            );
+            QueryClient eqlClient = new PITAwareQueryClient(eqlSession);
+            List<Criterion<BoxedQueryRequest>> criteria = new ArrayList<>();
+
+            for (int i = 0; i < sequenceFiltersCount; i++) {
+                final int j = i;
+                criteria.add(
+                    new Criterion<>(
+                        i,
+                        new BoxedQueryRequest(
+                            () -> SearchSourceBuilder.searchSource().size(10).query(matchAllQuery()).terminateAfter(j),
+                            "@timestamp",
+                            emptyList(),
+                            emptySet()
+                        ),
+                        keyExtractors,
+                        tsExtractor,
+                        null,
+                        implicitTbExtractor,
+                        false
+                    )
+                );
+            }
+
+            SequenceMatcher matcher = new SequenceMatcher(sequenceFiltersCount, false, TimeValue.MINUS_ONE, null, eqlCircuitBreaker);
+            TumblingWindow window = new TumblingWindow(eqlClient, criteria, null, matcher);
+            window.execute(
+                wrap(
+                    p -> { fail("Search succeeded despite PIT failure"); },
+                    ex -> { assertEquals(PIT_EXCEPTION_MESSAGE, ex.getMessage()); }
```

Comment:
`PIT_EXCEPTION_MESSAGE` can be inlined. Why the redirection?

Reply:
PIT_EXCEPTION_MESSAGE is used in two different places, that's why it's defined as a constant